src/main/java/frc/robot/subsystems/swerve/swervestatehelpers/AimAssistUtils.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems.swerve.swervestatehelpers;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.subsystems.swerve.SwerveConstants;
+import frc.robot.subsystems.swerve.SwerveMath;
+
+import java.util.function.Supplier;
+
+import static frc.robot.subsystems.swerve.SwerveMath.getDriveMagnitude;
+
+public class AimAssistUtils {
+
+    public static ChassisSpeeds getRotationAssistedSpeeds(ChassisSpeeds inputSpeeds,
```

Comment:
```suggestion
    public static ChassisSpeeds getRotationAssistedSpeeds(ChassisSpeeds speeds,
```
can be chassis speeds